src/types.deno.ts

```diff
@@ -87,7 +87,13 @@ export class InputFile {
         if (!(file instanceof URL)) return undefined;
         return basename(file.pathname) || basename(file.hostname);
     }
-    async [toRaw](): Promise<
+    /**
+     * Internal method. Do not use.
+     *
+     * Converts this instance into a binary representation that can be sent to
+     * the Bot API server in the request body.
+     */
+    async toRaw(): Promise<
         Uint8Array | Iterable<Uint8Array> | AsyncIterable<Uint8Array>
```

Comment:
Ideally, there should be a `.stream()` method that always returns `AsyncIterable<Uint8Array>`, to match `Blob`.

Reply:
I will add this as soon as there's demand, thanks for the suggestion.